### ISSUE TITLE
cant change theme when window size is small

### ISSUE THREAD

**Issue by zsytssk** [NvChad/base46#204]
**Expected behavior**
cant change theme when window size is small

**Screenshots**
![20230624-141208](https://github.com/NvChad/NvChad/assets/2304648/097db460-a62f-45cf-b48e-38e8eed594cc)

**Desktop (please complete the following information):**
 - Operating System: MAC M1 13 pro, 13.2.1
 - Terminal: Alacritty
 - Version of Neovim: 0.9.0

**Comment by siduck** [comment]
https://github.com/NvChad/base46/assets/59060246/3d957cee-1659-426e-b0f0-8a084277f966


unable to reproduce, please update all of your plugins and nvchad, i have fixed this issue long ago

**Comment by zsytssk** [comment]
@siduck my nvchad and all plugin is latest, you can see it in this video
![20230625-085412](https://github.com/NvChad/base46/assets/2304648/48463477-0120-4fee-b7a1-b4c4317a3d7c)

**Comment by siduck** [comment]
strange, i cant reproduce this at all... as well as others

**Comment by zsytssk** [comment]
@siduck Mybe is about the system I use，MAC M1 13 pro, macos Ventura 13.2.1

**Comment by LexeyKhom** [comment]
> simplescreenrecorder-2023-06-24_12.16.04.mp4
> 
> unable to reproduce, please update all of your plugins and nvchad, i have fixed this issue long ago

Try using the arrows (up and down) instead of searching. Using search it works fine

**Comment by siduck** [comment]
> > simplescreenrecorder-2023-06-24_12.16.04.mp4
> > unable to reproduce, please update all of your plugins and nvchad, i have fixed this issue long ago
> 
> Try using the arrows (up and down) instead of searching. Using search it works fine

yes, i thought users would just use C-n/ C-p to cycle in telescope
